did_resolver_sov/src/resolution/resolver.rs

```diff
@@ -31,11 +31,14 @@ impl DidSovResolver {
 
 #[async_trait]
 impl DidResolvable for DidSovResolver {
+    // TODO: Change to ExtraFields from Sovrin-specific DDO wrapper
```

Comment:
Wouldn't this be in scope of the PR? Isn't this quite important for integration?

Reply:
Sovrin-specific DDO wrapper is implemented in #871 based on this PR; it is impossible to use the extra fields before they are implemented.